lib/jenkins_pipeline_builder/extensions/publishers.rb

```diff
@@ -108,6 +108,21 @@
       end
     end
   end
+
+  version '2.0.0' do
```

Comment:
Were no new options added to the job config?

Reply:
Not that I saw. I used https://p2-cd-jenkins.ctct.net/job/HelloWorldJava-Pipeline/config.xml as my template after making sure it was on 2.0.0. Was that the correct thing to do?